### DISCUSSION TITLE
Complete documentation

### DISCUSSION THREAD

**Discussion by ggeldenhuis** [k3s-io/k3s#10580] [Q&A]
Hi
I am looking for more complete documentation that details how to configure a k3s cluster. 

It is not really clear from the existing documentation how to do anything other than a multi agent cluster. 

There is some references strewn around regarding setting up a multi master cluster but nothing definitive or complete. 

Related Projects appears to be the answer for setting up multi server clusters but I am keen on understanding how things are configured rather than installing yet another tool.

**Comment by ggeldenhuis** [comment]
I wanted to clarify a bit. The documentation is relatively decent but a bit jumbled. For example, I wanted to know how to configure a node to not run workloads which I would logically look for in the configuration options in installation section. However this particular bit of information is in the High Availability External DB section. So perhaps the question is not about "complete" documentation but perhaps if there is a case to be made to re-organise the documentation a bit, something I might be potentially interested in to help with.

**Comment by shakibamoshiri** [comment]
what / which part is unclear ?

to me, it is clear I had no issue reading and doing what ever I want
I spent a few days reading all subjects on [https://docs.k3s.io/](https://docs.k3s.io/) and did some testing
Any thing you need is there
